components/MessageError/index.js

```diff
@@ -0,0 +1,44 @@
+import { useRouter } from "next/router";
+import styled from "styled-components";
+import SVGIcon from "../Icons";
+
+export default function MessageError() {
+  const router = useRouter();
+
+  return (
+    <MessageContainer>
+      <MessageHeadline>Fehler beim Laden</MessageHeadline>
+      <MessageButton onClick={router.reload}>
```

Comment:
Every button should have a type for better readability. As the default type of every button is "submit" and you should add "button" to prevent behavior you do not want.

Reply:
fixed